pkg/binlog/reader/file.go

```diff
@@ -99,7 +99,7 @@ func (r *FileReader) StartSyncByPos(pos gmysql.Position) error {
 	defer r.mu.Unlock()
 
 	if r.stage != common.StageNew {
-		return errors.Errorf("stage %s, expect %s, already started", r.stage, common.StageNew)
+		return terror.ErrReaderAlreadyRunning.Generate(r.stage, common.StageNew)
```

Comment:
`terror.ErrReaderAlreadyRunning` do not accept other args? (and no args used in _pkg/streamer/reader.go_)